routes/auth.js

```diff
@@ -72,4 +72,21 @@ router.get('/version', function (req, res, next) {
     });
 });
 
+router.get('/session-ttl',async function(req, res){
+    let response
+    if (req.session){
+        const currentTime = Date.now();
+        const sessionExpiration = new Date(req.session.cookie.expires);
+        const sessionExpirationTime = sessionExpiration.valueOf();
+        if(!sessionExpiration || currentTime > sessionExpirationTime){
+            response = {ttl:0}
+        }else{
+            response = {ttl: sessionExpiration.valueOf() - currentTime}
```

Comment:
We need to return this time in seconds but I think this currently returns this in milliseconds